### ISSUE TITLE
CompRx Variational Autoencoder

### ISSUE THREAD

**Issue by StrongTanisha** [StrongResearch/isc-demos#64]
### Source / repo
https://github.com/StanfordMIMI/CompRx
https://github.com/StanfordMIMI/CompRx/blob/main/comprx/models/components/autoencoder_vq.py

### Model description
[DESCRIPTION]

### Dataset
[DATASET]

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]

**Comment by StrongTanisha** [comment]
Roadblock - requirements in the repo were incorrect. Not much instruction on how to run it
